### DISCUSSION TITLE
Add Hyprdots to the AUR

### DISCUSSION THREAD

**Discussion by hchasens** [prasanthrangan/hyprdots#889] [Ideas]
Since, so far, Hyprdots has only supported Arch, it makes sense that it should be added to the AUR. This should streamline installation for end users. (especially if they use an AUR package manager).

**Comment by kRHYME7** [comment]
Hello, Yeps Hyprdots is already in the AUR. 😁  it's the ``` hyprdots-ctl ``` package. It works as an installer of hyprdots. 
https://github.com/notifications?query=repo%3AkRHYME7%2FHyprdots-ctl 
I did not add the dependencies as it would broke user preferences.

**Comment by hchasens** [comment]
Apologies! I wasn't aware that hyprdots-ctl was an installer.

**Reply by kRHYME7** [reply]
Hahaha can't make a docu yet. But I am planning to make it more interactive. For now it will just use the native way from this repository. 



To manage updates from repo  ``` Hyprdots update ``` then to apply  use ``` Hyprdots restore [options ]```

**Reply by hchasens** [reply]
Might I suggest adding that to the `README.md`?

**Reply by kRHYME7** [reply]
Soon...

**Reply by hchasens** [reply]
lol. Apologies for my eagerness. Can't wait to see it!

**Reply by kRHYME7** [reply]
I'm really eager to add it too.😂 I only need a single install.sh for AUR helper. Then install hyprdot-ctl. Invoking ``` Hyprdots ``` will trigger an installation if it can't find the essential files.


BTW, Have you tried the hyprland-git version? I think it has rendering issues.

**Reply by hchasens** [reply]
No, I haven't. Strangely the normal Hyprland package is a version ahead of the git AUR. Not sure how that happened. I generally use packages unless I have a good reason to use the AUR since Arch does a good job of keeping their packages up to date.

**Reply by kRHYME7** [reply]
That Package Ver on the AUR hyprland-git is not bumped as it will always rebuild  if there are changes in commit. Therefore the hyprland-git still is the latest version.

**Reply by hchasens** [reply]
I see. Sometimes there's an issue with the PKGBUILD during version changes and the maintainer gets to it later than the official arch repo. I didn't know it had to be bumped by a build for the version change to show on the AUR though. That makes me trust -git AURs a bit more! Thank you for the new information :)

**Reply by kRHYME7** [reply]
What happens is the static package ver will be displayed.  (Can be too old)

Then the maintainer will add a function that overrides the static version to the latest tag available.  This way the package version will change, then it will rebuild. (it also tracks the latest tag installed as package version, so no rebuilds on identical git tag.) 

BTW, I found out my waybar is causing this issue, it messes up layers for hyprland. 
![240220_18h57m14s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/53417443/1cf384ee-1d58-4ae7-b276-01aae9b13bd9)
![240220_18h58m04s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/53417443/59709aa1-e030-4a0a-99c6-6029696e472f)
This 
![240220_18h58m16s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/53417443/7ba6e4f4-c527-42c4-84bb-43097b99ae37)

Problem won't occur if I hide waybar on the layers, (SIGUSR). If ever you found something related, care to ping me? Thank you!

**Reply by hchasens** [reply]
Sorry I'm not sure why waybar would do this. Does the issue persist with other configurations of waybar? If you use the default conf does it persist? This is very odd and I'm not sure how much help I'll be.

**Reply by kRHYME7** [reply]
@hchasens It's fine now after some maturity of the git version of hyprland. 
I can confirm that it's something to do with waybar because when I hide waybar the issue is gone. Only manage to replicate it on another Laptop with intel + nvidia dGPU. Maybe this is something to do with layers rendering.  I tested it on an isolated user with hyprland and waybar with default config. It's fixed now, so I can't replicate . (Also reverting to hyprland stable fixes it though)
